help/help.md

```diff
@@ -1,29 +1,29 @@
 % HAPROXY(1) Container Image Pages
 % Petr Hracek
-% March 2, 2017
+% September 22, 2017
 
 # NAME
-haproxy - HAProxy reverse proxy for high availability environments.
+{{ spec.envvars.name }} - {{ spec.short_description }}
 
 # DESCRIPTION
-HAProxy is a TCP/HTTP reverse proxy which is particularly suited for high availability environments.
+{{ spec.description }}
 
 The container itself consists of:
-    - fedora/26 base image
-    - haproxy RPM package
+    - {{ config.docker.from }} base image
```

Comment:
I would probably still prefer to have the complete name, but this is good enought